Client/src/Components/product-overview/StyleSelector.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from "react";
+import DataContext from "../context";
+
+function StyleSelector() {
+  const data = useContext(DataContext);
+  const { styleIndex, styles } = data;
+  const sizes = [];
+  const quantitySelector = (max) => {
+    const quantitiesSelect = [];
+    if (max > 15) {
+      max = 15;
+    }
+    for (let i = 1; i < max + 1; i += 1) {
+      quantitiesSelect.push(i);
+    }
+    return quantitiesSelect;
+  };
+  const getSizes = () => {
+    Object.keys(styles[styleIndex].skus).forEach((key) => {
+      sizes.push({
+        size: styles[styleIndex].skus[key].size,
+        quantity: quantitySelector(styles[styleIndex].skus[key].quantity),
+      });
+    });
+  };
+  getSizes();
+  const [currentSelect, setCurrentSelect] = useState(0);
```

Comment:
Probably better to have useState functions declared at the top, but its up to you